src/sentry/api/event_search.py

```diff
@@ -78,15 +78,15 @@
 has_filter = negation? "has" sep (search_key / search_value)
 is_filter  = negation? "is" sep search_value
 
-aggregate_key    = key open_paren function_arg* closed_paren
+aggregate_key    = key open_paren spaces function_args? spaces closed_paren
 search_key       = key / quoted_key
 search_value     = quoted_value / value
 value            = ~r"[^()\s]*"
 in_value         = ~r"[^(),\s]*(?:[^\],\s)]|](?=]))"
 numeric_value    = ~r"([-]?[0-9\.]+)([kmb])?(?=\s|\)|$|,|])"
 boolean_value    = ~r"(true|1|false|0)(?=\s|\)|$)"i
 key              = ~r"[a-zA-Z0-9_\.-]+"
-function_arg     = spaces key? comma? spaces
```

Comment:
The problem here was not only that you could have args without commas. But that this is a token that could possibly never consume. In the JS version of the peg parser, that is a loop error.